app/models/manageiq/providers/azure/event_catcher_mixin.rb

```diff
@@ -28,8 +28,8 @@ def parse_vm_ref(event)
     _providers,
     provider,
     object_class,
-    object_name = event["resourceId"].downcase.split("/")
+    object_name = event["resourceId"].split("/")
```

Comment:
@djberg96 - because I don't want to downcase the obect_name (vm name), it has to match the VM ems_ref set in the refresh_parser.